src/expire-tiles.cpp

```diff
@@ -166,6 +187,7 @@ void expire_tiles::from_line_segment(geom::point_t const &a,
                                      geom::point_t const &b,
                                      expire_config_t const &expire_config)
 {
+    log_info("void expire_tiles::from_line_segment(geom::point_t const &a, geom::point_t const &b, expire_config_t const &expire_config)");
```

Comment:
do we need all the log infos ?

Reply:
used only for debugging and removed eventually in 6c68ead630575bc2545cd668c6e6dc7b6d4772f9